README.md

```diff
@@ -116,11 +124,16 @@ PING  (142.250.217.78) 56(84) bytes of data.
 rtt min/avg/max/mdev = 13.985/14.779/15.886/0.807 ms
 ```
 
-To select multiple locations you can use a comma as a delimiter. You can mix and match the location types without issues. 
-We also set a limit of 4 probes to get 1 answer per location, otherwise the default limit of 1 would result in a random result from one of the four locations.
+> [!TIP]
+> We recommend reading our [tips and best practices](https://github.com/jsdelivr/globalping#best-practices-and-tips) to learn more about defining locations effectively!
+
+#### Define multiple locations and basic flags
+Select multiple locations for running a command by using a comma `,` as a delimiter. When doing so, make sure to also specify the number of tests to run with the `--limit` flag.
+For example, if you want to run ping from four different locations, add `--limit 4` to make sure you get one test result per location. Otherwise, the default limit of 1 will be selected, resulting in a random result from one of the four locations.
 
-Last we use the `--latency` parameter to only get the summary latency data instead of the full raw ping output. 
+Let's also introduce you to another handy flag: If you're only interested in the latency metrics for your tests and not the full raw output, add `--latency` to your command.
```

Comment:
Lets revert, the CLI is aimed at advanced users and this sounds too "newbie" focused

Reply:
Updated this to be more like the original version